backend/btrixcloud/crawlconfigs.py

```diff
@@ -108,9 +108,31 @@ class CrawlConfigIn(BaseModel):
     crawlTimeout: Optional[int] = 0
     scale: Optional[conint(ge=1, le=MAX_CRAWL_SCALE)] = 1
 
+    # for now, until frontend is changed
     oldId: Optional[UUID4]
```

Comment:
I guess can remove this now with frontend changes

Reply:
removed